packages/react/src/VideoPlayer/VideoPlayer.module.css

```diff
@@ -96,50 +118,49 @@
 /* 5. Controls                                                */
 /* ---------------------------------------------------------- */
 
-.VideoPlayer__controls {
-  transition: var(--brand-VideoPlayer-transition);
-  bottom: 0;
-  position: absolute;
-  z-index: 2;
-  width: 100%;
-  pointer-events: none;
-  opacity: 1;
-  transition: opacity 2s ease-in-out;
-}
-
-.VideoPlayer__controls--hidden {
-  opacity: 0;
-}
-
 .VideoPlayer__controlsBar {
-  display: flex;
-  gap: var(--base-size-16);
+  min-height: var(--brand-VideoPlayer-controls-min-height);
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-flow: column;
+  justify-content: space-between;
   align-items: center;
+  gap: var(--base-size-16);
   border-bottom-right-radius: var(--brand-borderRadius-medium);
   border-bottom-left-radius: var(--brand-borderRadius-medium);
-  left: 0;
   width: 100%;
-  background: var(--brand-videoPlayer-controls-bgColor);
   padding: var(--base-size-12) var(--base-size-16);
   pointer-events: all;
   opacity: 1;
 }
 
-.VideoPlayer__controlsBar--fade {
-  transition: opacity var(--brand-animation-duration-default) var(--brand-animation-easing-default);
-  opacity: 0;
+.VideoPlayer__controlsBar__row1,
+.VideoPlayer__controlsBar__row2 {
+  display: flex;
+  align-items: center;
+  gap: var(--base-size-16);
+}
+
+@media (max-width: 30rem) {
```

Comment:
30rem isn't a breakpoint that we use anywhere else, however it makes sense for this particular component.

This is the point at which the single row of controls breaks to two rows

Reply:
Our `small` breakpoint is `34rem`, could that work?